src/classes/ERR_ExceptionHandler.cls

```diff
@@ -56,11 +57,16 @@ public with sharing class ERR_ExceptionHandler {
     */
     public static void beautifyExceptionMessage(Exception ex) {
         String message = ex.getMessage();
+        String typeName = ex.getTypeName();
```

Comment:
Just for my understanding, is it that in this case the identifier for the exception is in the type instead of the message?